process/smartContract/process.go

```diff
@@ -1699,11 +1703,12 @@ func createBaseSCR(
 	outAcc *vmcommon.OutputAccount,
 	tx data.TransactionHandler,
 	txHash []byte,
+	transferNonce uint64,
 ) *smartContractResult.SmartContractResult {
 	result := &smartContractResult.SmartContractResult{}
 
 	result.Value = big.NewInt(0)
-	result.Nonce = outAcc.Nonce
+	result.Nonce = outAcc.Nonce + transferNonce
```

Comment:
This should be `result.Nonce = transferNonce`, without `outAcc.Nonce`.

Reply:
the idea was to always start from the account's nonce and increment from there. Suggested by @sasurobert